main.cpp

```diff
@@ -125,8 +135,15 @@ int main(int argc, char *argv[]) {
 
 	std::atexit(destroy_texture);
 
-	Uint32 pixels[WIDTH * HEIGHT] = {0};
-	int pixel_index = 0; // TODO: Remove this when connecting the pixel display to the emulator core.
+	u8 pixels[NUM_PIXELS] = {};
```

Comment:
Mingw also complained about `{0}`. Good news is that in C++, unlike C, `{}` initializes the array to all zeros anyway.